projects/fsastorefront-e2e-cypress/cypress/integration/insurance/auto-claim-update.e2e-spec.ts

```diff
@@ -0,0 +1,134 @@
+      checkout.waitConsent();
+    });
+
+    it('Should complete auto checkout', () => {
+      checkout.startInsuranceCheckout('Auto');
+      auto.populateAutoAnnuallyTesla();
+      auto.populateMainDriverInfo();
+      cy.get('[name=noOfDrivers]').select('0');
+      checkout.clickContinueButton();
+      autoIntegration.selectAutoBronze();
+      auto.checkOptionalProductsBronze();
+      checkout.clickContinueButton();
+      checkout.checkCheckoutStep('Your Auto Insurance', '7');
+      checkout.checkPersonalDetailsPage();
+      auto.populatePersonalDetails();
+      auto.populateVehicleDetails();
+      auto.populateMainDriverData();
+      checkout.clickContinueButton();
+      checkout.checkCheckoutStep('Your Auto Insurance', '7');
+      checkout.checkProgressBarInsurance();
+      checkout.checkAccordions('generalQuoteAccordions');
+      checkout.checkBackAndContinueButtons();
+      checkout.clickContinueButton();
+      checkout.ConfirmBindQuote();
+      checkout.checkAccordions('generalQuoteAccordions');
+      checkout.clickContinueButton();
+      selectPaymentMethodInvoice();
+      checkout.clickContinueButton();
+      checkout.placeOrderOnFinalReview();
+      checkout.checkOrderConfirmation();
+      //waiting for replication process to be completed
+      cy.wait(200000);
+      myPolicies.checkMyPoliciesPage();
+    });
+
+    it('Should submit a claim - Glass Damage', () => {
+      autoIntegration.startFnolCheckout('Bronze');
+      autoIntegration.waitForIncidentInfoForm();
+      fnol.checkFNOLCheckoutPage();
+      fnol.checkFNOLSteps();
+      fnol.populateIncidentInformationStep('Glass Damage');
+      checkout.clickContinueButton();
+      fnol.populateIncidentReportStep();
+      cy.get('[name=incidentAction]').select('Repair broken glass');
+      cy.get('[name=amountRequested]').type('4500');
+      fnol.checkDownloadButton();
+      checkout.checkBackAndContinueButtons();
+      checkout.clickContinueButton();
+      autoIntegration.waitForGeneralInfoForm();
+      fnol.checkFNOLCheckoutPage();
+      fnol.populateGeneralInformationStep();
+      checkout.clickContinueButton();
+      fnol.checkFNOLCheckoutPage();
+      fnol.checkSummaryPage();
+      cy.get('.action-button').should('contain.text', 'Back');
+      cy.get('.primary-button').should('contain.text', 'Submit');
+      autoIntegration.checkIncidentInformationAccordion('AutoGlassDamage');
+      fnol.checkIncidentReportAccordion('5');
+      fnol.checkGeneralInformationAccordion();
+      cy.get('.primary-button').should('be.visible');
+      cy.get('.primary-button').contains('Submit').click({ force: true });
+      fnol.checkConfirmationPage();
+      cy.get('.SiteLogo').should('be.visible').click();
+      homepage.checkPageElements();
+    });
+
+    it('Should check documents page', () => {
+      checkMyDocuments('1');
+      cy.get('span.td-value').contains('Claim.pdf');
```

Comment:
Is there a need for `span` in this selector?

Reply:
removed